README.md

```diff
@@ -1 +1,15 @@
 # hw01
+
+## Self Introduction
+My name is Yuwen Xiong. I am a second-year master student at **UChicago Harris School of Public Policy**. Below is the photo of mine:
+
+![Photo of Yuwen](LinkedIn-profile-smaller.jpg)
```

Comment:
The picture turns out really nice, but maybe adjust the size of it so it's not too big.